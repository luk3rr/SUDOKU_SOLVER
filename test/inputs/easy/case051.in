503020000 400800000 002159000 039002000 000630800 000000040 000000780 018200400 045900600 
