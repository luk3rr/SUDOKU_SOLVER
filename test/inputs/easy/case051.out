Solving the following grid:
5 0 3 | 0 2 0 | 0 0 0 
4 0 0 | 8 0 0 | 0 0 0 
0 0 2 | 1 5 9 | 0 0 0 
------+-------+------
0 3 9 | 0 0 2 | 0 0 0 
0 0 0 | 6 3 0 | 8 0 0 
0 0 0 | 0 0 0 | 0 4 0 
------+-------+------
0 0 0 | 0 0 0 | 7 8 0 
0 1 8 | 2 0 0 | 4 0 0 
0 4 5 | 9 0 0 | 6 0 0 


Solution found :')

5 7 3 | 4 2 6 | 9 1 8 
4 9 1 | 8 7 3 | 5 6 2 
6 8 2 | 1 5 9 | 3 7 4 
------+-------+------
8 3 9 | 7 4 2 | 1 5 6 
2 5 4 | 6 3 1 | 8 9 7 
1 6 7 | 5 9 8 | 2 4 3 
------+-------+------
9 2 6 | 3 1 4 | 7 8 5 
7 1 8 | 2 6 5 | 4 3 9 
3 4 5 | 9 8 7 | 6 2 1 

Algorithm: BFS
Total time: 48 ms
Total expanded states: 8071
